.claude/commands/generate-prp.md

```
@@ -1,69 +1,636 @@
+
+        📄 File: PRPs/add-user-authentication-jwt.md
+        ⏱️  Estimated: 6 days
+
+### Multiplayer Example:
+```
+User: /generate-prp Add lobby system with room matchmaking
+
+Claude: 🤖 Generating PRP for "Add lobby system with room matchmaking"...
+        🔍 Detected: Multiplayer feature (lobby, matchmaking keywords)
+
+        📝 Creating boilerplate...
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)